src/igvfd/audit/biosample.py

```diff
@@ -0,0 +1,36 @@
+from snovault.auditor import (
+    audit_checker,
+    AuditFailure,
+)
+from .formatter import (
+    audit_link,
+    path_to_text,
+)
+
+
+@audit_checker('Biosample', frame='object')  # this audit requires metadata in other objects - donor taxa
+def audit_biosample_taxa_check(value, system):
+    '''Flag any biosample with populated donors, but without taxa meaning the donors are mixed taxas.'''
+
+    if 'donors' in value:  # if donors property is populated in sample object
+        sample_id = value['@id']  # get the sample id
+        donor_ids = value.get('donors')  # get the list of donors
+        # create dictionary for storing the taxa (key) and a list of donors (value) associated with that taxa
+        taxa_dict = {}
+        detail = []  # create detail list for collecting error details
```

Comment:
i think you can move detail under (to the place where a problem >1 was detected)